.github/workflows/framework-kind.yml

```
@@ -0,0 +1,16 @@
+name: Framework
+
+on:
+  push:
+    tags:
+      - v0.**
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  fw-kind-tests:
+    name: KinD
+    uses: stolostron/governance-policy-framework/.github/workflows/kind.yml@main
+    with:
+      hub_only_component: 'true'
```

nit: trailing newline
```suggestion
      hub_only_component: 'true'

```